src/BloomExe/Publish/Video/PublishAudioVideoAPI.cs

```diff
@@ -391,8 +391,8 @@ public void AbortMakingVideo()
 		{
 			if (_recordVideoWindow != null)
 			{
-				_recordVideoWindow.Close();
 				_recordVideoWindow.Cleanup();
+				_recordVideoWindow.Close();
```

Comment:
I think this was a bug that has been there for at least several versions. Null reference exception if you close it first and then try to clean it up